CMakeLists.txt

```diff
@@ -65,17 +67,6 @@ if(CMAKE_Platform)
   else()
     message("Platform '${CMAKE_Platform}' configuration file does not exist")
   endif()
-
```

Comment:
We need to make sure that this patch is no longer required, or apply it in hpc-stack.

Reply:
This patch is now applied in hpc-stack, it was still necessary.